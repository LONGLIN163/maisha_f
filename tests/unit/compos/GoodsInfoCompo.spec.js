import {mount, createLocalVue} from '@vue/test-utils'
import GoodsInfoCompo from '@/components/compos/GoodsInfoCompo.vue'

import Vue from 'vue'
const Vant=require("vant")
const {Lazyload}=Vant
Vue.use(Lazyload)

const localVue = createLocalVue();
localVue.use(Vant); 

describe('GoodsInfoCompo', () => {
  let wrapper;
  const fakedata = {
    goodsImage:"https://m.media-amazon.com/images/I/81uKjZIuWsL._AC_SL1500_.jpg",
    goodsName: "Granini sin azúcar añadido zumo 100 % frutas",
    goodsPrice: 3.9,
    goodsId: "fb0f913950944b66a97ae262ad14609a"
  }
  beforeEach(() => {
    wrapper = mount(GoodsInfoCompo,{
      localVue,
      propsData: fakedata
    })
  })

  it('should renders GoodsInfoCompo', () => {
    expect(wrapper).toMatchSnapshot()
  })
})