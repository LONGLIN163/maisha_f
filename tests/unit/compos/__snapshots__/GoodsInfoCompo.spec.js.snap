// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoodsInfoCompo should renders GoodsInfoCompo 1`] = `
<div class="goods-info">
  <div class="goods-image"><img width="90%" data-src="https://m.media-amazon.com/images/I/81uKjZIuWsL._AC_SL1500_.jpg" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" lazy="loading"></div>
  <div class="goods-name">Granini sin azúcar añadido zumo 100 % frutas</div>
  <div class="goods-price">3.90 €</div>
</div>
`;
